python/servo/package_commands.py

```diff
@@ -315,8 +318,14 @@ def package(self, release=False, dev=False, android=None, debug=False, debugger=
             print("Packaged Servo into " + path.join(dir_to_msi, "Servo.msi"))
 
             print("Creating ZIP")
-            shutil.make_archive(path.join(dir_to_msi, "Servo"), "zip", dir_to_temp)
-            print("Packaged Servo into " + path.join(dir_to_msi, "Servo.zip"))
+            zip_path = path.join(dir_to_msi, "Servo.zip")
+            import zipfile
```

Comment:
It would be best to create the zip archive in a deterministic way; it's probably easiest to fold this into `archive_deterministically` or make a similar helper method. See https://reproducible-builds.org/docs/archives/ for a list of major considerations.

That method also has some other important features, like using `with` statements to close files and automatically un-`cd`, and atomic renaming of the output to prevent half-finished archives from being uploaded.